lib/commands/context.js

```diff
@@ -61,6 +61,13 @@ commands.setContext = async function setContext (name) {
   this.curContext = name;
 };
 
+commands.mobileGetContexts = async function mobileGetContexts () {
+  const opts = _.cloneDeep(this.opts);
```

Comment:
should we really clone all opts here? we know which options are needed, so it would be fine to just create options objections here based on opts values

Reply:
Ah, no. So I just created a opt object there.